lib/elixir/lib/option_parser.ex

```diff
@@ -71,6 +71,9 @@ defmodule OptionParser do
   Note that you should only supply the `:switches` or the`:strict` option.
   If you supply both, an `ArgumentError` exception will be raised.
 
+    iex> OptionParser.parse(["--source", "lib"], switches: [source: :string], strict: [source: :string])
+    ** (ArgumentError) :switches and :strict cannot be given together
+
```

Comment:
Are we adding anything through this example? This behaviour is noted in the line above and I think that's quite clear so maybe we can skip adding this 🤔